src/main/java/com/adyen/service/resource/payment/Authorise3DS2.java

```diff
@@ -0,0 +1,13 @@
+package com.adyen.service.resource.payment;
+
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.service.Resource;
+
+public class Authorise3DS2 extends Resource {
```

Comment:
is this used?

Reply:
Based the documentation it is used 
https://docs.adyen.com/api-explorer/#/Payment/v40/authorise3ds2